### DISCUSSION TITLE
error upon attempting to boot hyprland from sddm dm

### DISCUSSION THREAD

**Discussion by Bluerazor41** [end-4/dots-hyprland#766] [Q&A]
Im not quite sure whats happened
![20240823_232415](https://github.com/user-attachments/assets/49853daa-45ba-492a-a480-8fee995c80b6)

**Comment by XanoodlesThaBat** [comment]
Did you manage to fix it somehow?

**Comment by end-4** [comment]
does running `Hyprland` in tty work?
